app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -3,11 +3,11 @@
 
   <% if follow_request %>
     <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request, id: dom_id(follow_request), class: "btn btn-outline-secondary", remote: true, method: :delete do %>
         Un-request
       <% end %>
     <% elsif follow_request.accepted? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request, id: dom_id(follow_request), class: "btn btn-outline-secondary", remote: true, method: :delete do %>
```

Comment:
Elegant way of appointing ID to a follow_request button to get it handled by AJAX